splitting_months.py

```diff
@@ -0,0 +1,45 @@
+main_file = open("http_access_log.txt")
+
+oct = open("October.log", "w")
```

Comment:
It is common practice to use the close() function when opening files to be written 
Here is the documentation for that method: https://www.tutorialspoint.com/python/file_close.htm